python/ray/data/_internal/block_batching/block_batching.py

```diff
@@ -27,17 +29,14 @@ def nullcontext(enter_result=None):
         yield enter_result
 
 
-PREFETCHER_ACTOR_NAMESPACE = "ray.dataset"
-
-
 def batch_block_refs(
     block_refs: Iterator[ObjectRef[Block]],
     *,
     stats: Optional[Union[DatasetStats, DatasetPipelineStats]] = None,
     prefetch_blocks: int = 0,
     clear_block_after_read: bool = False,
     batch_size: Optional[int] = None,
-    batch_format: Optional[str] = "default",
+    batch_format: str = "default",
```

Comment:
This should be reverted to Optional[str] here and elsewhere, right?

Reply:
updated